config/application.rb

```diff
@@ -14,5 +14,9 @@ class Application < Rails::Application
     # Settings in config/environments/* take precedence over those specified here.
     # Application configuration should go into files in config/initializers
     # -- all .rb files in that directory are automatically loaded.
+    logger           = ActiveSupport::Logger.new(STDOUT)
```

Comment:
This bit made logs more useful running behind Puma.  Otherwise, nothing gets logged to stdout and it's not helpful for anyone.